### PR TITLE
Update .editorconfig and fix formatting for renovate.json

### PR DESCRIPTION
**Pull request by Maxr1998**

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -3,5 +3,25 @@
...
+				]
+			},
+			{
+				"groupName": "detekt",
+				"matchPackagePrefixes": [
+					"io.gitlab.arturbosch.detekt"
+				]
+			}
+		]
+	}
```
Comment (nielsvanvelzen) [inline renovate.json:26]: These should go to:
https://github.com/jellyfin/.github/blob/master/renovate-presets/gradle.json

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: renovate.json (hunk 1 above)
Comment (Maxr1998) [reply renovate.json:26]: I literally didn't find the GitHub repo earlier...

But are we sure we want to enable that for all repos?

Comment (nielsvanvelzen) [reply renovate.json:26]: The preset is used for jellyfin-android, jellyfin-androidtv and jellyfin-sdk-kotlin so yes.

Comment (Maxr1998) [reply renovate.json:26]: Alright. Superseded by jellyfin/.github#11 then.

Comment (Maxr1998) [reply renovate.json:26]: I'll use this PR to fix the formatting then, I saw the `.github` repo uses 2 spaces too.

File: .editorconfig [hunk 2]
```diff
@@ -18,3 +18,8 @@ trim_trailing_whitespace = true
 ij_kotlin_code_style_defaults = KOTLIN_OFFICIAL
 ij_kotlin_name_count_to_use_star_import = 999
 ij_kotlin_name_count_to_use_star_import_for_members = 999
+
+[renovate.json]
```
Comment (nielsvanvelzen) [inline .editorconfig:22]: Guess this could be *.json

Comment (Maxr1998) [reply .editorconfig:22]: Oh right, now that I changed it. Will fix.

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**